CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,26 @@
 # Changelog 1.11
 
+## 1.11.2 - 2023-05-29
+
+### Changed
+
+- Update Open vSwitch to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Bump up whereabouts to v0.6.1. ([#4988](https://github.com/antrea-io/antrea/pull/4988), [@hjiajing])
+
+### Fixed
+
+- Fix the Antrea Agent crash issue on Windows by running modules that rely on Services after AntreaProxy is ready. ([#4946](https://github.com/antrea-io/antrea/pull/4946), [@tnqn])
+- [Multicast] Clone remoteMembers when creating an updated GroupMemberStatus . ([#4903](https://github.com/antrea-io/antrea/pull/4903), [@ceclinux])
+- Make FQDN NetworkPolicy work for upper case DNS. ([#4934](https://github.com/antrea-io/antrea/pull/4934), [@GraysonWu])
```

Comment:
It's duplicated with line 20. You may update this with the date after all tests passed.